azure-pipelines.yml

```diff
@@ -182,6 +181,41 @@ jobs:
     parameters:
       artifact: sw_build
 
+# This is a second software build currently building with Clang, to test out the
+# compiler migration. It is a copy of `sw_build` with `continueOnError: true`
+# specified for the main tasks, and `meson_init.sh` configured with the clang
+# toolchain not the default toolchain.
+- job: sw_build_clang
+  displayName: Build Software (with Clang)
+  dependsOn: lint
+  condition: and(succeeded(), eq(dependencies.lint.outputs['DetermineBuildType.onlyDocChanges'], '0'))
+  pool:
+    vmImage: ubuntu-16.04
+  steps:
+  - template: ci/install-package-dependencies.yml
+  - bash: |
+      set -x
+      sudo util/get-toolchain.py \
+        --target-dir="${TOOLCHAIN_PATH}" \
```

Comment:
no need for curly braces here, but fine for consistency. If you feel like it, clean both targets up to remove them.